README.md

```diff
@@ -424,13 +424,13 @@ Other Style Guides
     // bad - no returned value means `acc` becomes undefined after the first iteration
     [[0, 1], [2, 3], [4, 5]].reduce((acc, item, index) => {
       const flatten = acc.concat(item);
-    });
+    }, []);
 
     // good
     [[0, 1], [2, 3], [4, 5]].reduce((acc, item, index) => {
       const flatten = acc.concat(item);
       return flatten;
-    });
+    }, []);
```

Comment:
this isn't needed, because a new array is returned each time, and when you omit the initial value, the first value is used.